redash/cli/data_sources.py

```diff
@@ -35,6 +35,15 @@ def list(organization=None):
             ds.id, ds.name, ds.type, ds.options.to_json()))
 
 
+@manager.command()
+def list_types():
+    print ("Enabled Query Runners:")
```

Comment:
The. Space. Before. The. Parentheses. Is. Killing. Me.

Reply:
Fixed. e9892b6